.github/workflows/testacc.yml

```diff
@@ -11,6 +11,9 @@ on:
       - main
     paths-ignore:
       - 'README.md'
+  schedule:
+    - cron: '30 21 * * 1'  # Runs every Monday at 2:30PST for stage environment
+    - cron: '30 21 * * 2,5'  # Runs every Tuesday and Friday at 2:30PST for dev environment
```

Comment:
this cron expression is also for 9:30 PM - is this or the comment wrong?

Reply:
same as above 9:30PM UTC is 2:30PM PST